src/plugins/discover/public/application/helpers/get_sharing_data.test.ts

```diff
@@ -46,10 +46,8 @@ describe('getSharingData', () => {
         ],
         "searchRequest": Object {
           "body": Object {
-            "_source": Object {
-              "includes": Array [],
-            },
```

Comment:
`_source.includes` now only gets set if you actually have some specified fields to include from source. the response from ES remains the same